### PR TITLE
Cplp 1549 partner network search issue

### PR DESCRIPTION
**Pull request by manojava-gk**

### CODE REVIEW

**Review by lavanya-bmw — CHANGES_REQUESTED** [review]
Please remove the console

File: cx-portal/src/components/pages/PartnerNetwork/index.tsx [hunk 1]
```diff
@@ -61,6 +61,11 @@ const PartnerNetwork = () => {
 
   const fetchAndApply = async (cData: any) => {
     //BPDM response does not has content attribute. Check for it and proceed
+    if (isContentPresent(cData) && cData.content.length === 0) {
+      console.log(cData)
+      setAllItems([])
+      return
```
Comment (lavanya-bmw) [inline cx-portal/src/components/pages/PartnerNetwork/index.tsx:67]: empty return?

**Comment by manojava-gk** [comment]
> Please remove the console

Updated.

File: cx-portal/src/components/pages/PartnerNetwork/index.tsx (hunk 1 above)
Comment (manojava-gk) [reply cx-portal/src/components/pages/PartnerNetwork/index.tsx:66]: yes. I do not want to run further in the method.

**Review by lavanya-bmw — APPROVED** [review]
lgtm

**Pull request merged**